static/js/pages/elections.js

```diff
@@ -30,7 +30,7 @@ var independentExpenditureColumns = [
         return {
           support_oppose_indicator: row.support_oppose_indicator,
           candidate_id: row.candidate_id,
-          is_notice: false,
+          is_notice: false
```

Comment:
This one was already here--I just took out a comma. We want the aggregates links to include `is_notice=false` in the URL, right?